src/pages/index.js

```diff
@@ -97,10 +173,42 @@ profilePencilBtn.addEventListener('click', () => {
   profileEditPopup.open();
 });
 
+// Edit Avatar Form
+
+const profileAvatarPopup = new PopupWithForm('#profile-avatar-modal', (newAvatar) => {
+  profileAvatarPopup.submitButtonState(true);
+  api
+    .updateAvatar(newAvatar)
+    .then((newAvatar) => {
+      userInfo.setAvatar({avatar: newAvatar.avatar});
+    })
+    .catch((err) => {
+      console.error(`Error: ${err}`)
```

Comment:
Good job that you `catch` possible errors at the end of server requests. You can simplify it with `.catch(console.error);` because `console.error` is meant to log errors